C/Coin-OR/Clp/build_tarballs.jl

```diff
@@ -45,19 +55,27 @@ fi
     --disable-dependency-tracking \
     --enable-shared \
     lt_cv_deplibs_check_method=pass_all \
-    --with-blas="-lopenblas" \
-    --with-lapack="-lopenblas" \
+    --with-blas \
+    --with-blas-lib="-L${libdir} ${LBT}" \
+    --with-lapack \
+    --with-lapack-lib="-L${libdir} ${LBT}" \
+    --with-coinutils \
     --with-coinutils-lib="-lCoinUtils" \
-    --with-osi-lib="-lOsi -lCoinUtils" \
-    --with-mumps-lib="-L${libdir} -ldmumps -lzmumps -lcmumps -lsmumps -lmumps_common -lmpiseq -lpord -lmetis -lopenblas -lgfortran -lpthread" \
```

Comment:
I guess we needed to expand on fortran versions because of this

Reply:
I don't think so because Clp will not linked to `libgfortran`.
It's MUMPS that is linked to `libgfortran`.
We need all these options only if we use static libraries.
Because we have a shared library `libdmumps.$dlext`, this library is already linked to `libmetis`, `libpord`, `libgfortran` etc...